incubator/metrics-http/src/test/java/io/aklivity/zilla/runtime/metrics/http/internal/HttpMetricsFactorySpiTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.metrics.http.internal;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.metrics.Metrics;
+
+public class HttpMetricsFactorySpiTest
+{
+    @Test
+    public void shouldCreate()
+    {
+        HttpMetricsFactorySpi httpMetricsFactorySpi = new HttpMetricsFactorySpi();
+        Configuration config = new Configuration();
+        Metrics metrics = httpMetricsFactorySpi.create(config);
+
+        assertThat(metrics.name(), equalTo("http"));
+        assertThat(metrics, instanceOf(HttpMetrics.class));
+    }
+}
```

Comment:
We usually don't test the `Spi` directly, instead we test indirectly via the factory, as you have done in `HttpMetricsFactoryTest`.

Suggest we remove this test class if feasible.